tests/integration/test_outcome.py

```diff
@@ -1037,7 +1036,6 @@ def test_outcomes_aggregate_dynamic_sampling(relay, mini_sentry):
         mini_sentry.captured_outcomes.get(timeout=1.2),
     ]
     assert mini_sentry.captured_outcomes.qsize() == 0
-    assert mini_sentry.captured_events.qsize() == 0
```

Comment:
What does the captured envelope contain here? It shouldn't contain any transactions IIUC.

Reply:
It contains the extracted metrics.